src/serac/physics/CMakeLists.txt

```diff
@@ -30,12 +30,7 @@ blt_add_library(
     DEPENDS_ON  mfem integrators numerics infrastructure physics_utilities physics_operators axom fmt mpi
     )
 
-target_include_directories(physics PUBLIC 
-    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/..>
-    $<INSTALL_INTERFACE:include>
-    )
```

Comment:
I might be missing something, but why was this removed?

Reply:
Everything in Serac inherits from `infrastructure` which has this PUBLIC.